validator/src/main.rs

```diff
@@ -1038,15 +1038,16 @@ pub fn main() {
         &format!("{}-{}", VALIDATOR_PORT_RANGE.0, VALIDATOR_PORT_RANGE.1);
     let default_genesis_archive_unpacked_size = &MAX_GENESIS_ARCHIVE_UNPACKED_SIZE.to_string();
     let default_rpc_max_multiple_accounts = &MAX_MULTIPLE_ACCOUNTS.to_string();
-    let default_rpc_pubsub_max_connections = PubSubConfig::default().max_connections.to_string();
-    let default_rpc_pubsub_max_fragment_size =
-        PubSubConfig::default().max_fragment_size.to_string();
-    let default_rpc_pubsub_max_in_buffer_capacity =
-        PubSubConfig::default().max_in_buffer_capacity.to_string();
-    let default_rpc_pubsub_max_out_buffer_capacity =
-        PubSubConfig::default().max_out_buffer_capacity.to_string();
+
+    // There params are no longer in use.
+    let default_rpc_pubsub_max_connections = 1000.to_string();
+    let default_rpc_pubsub_max_fragment_size = (50 * 1024).to_string();
+    let default_rpc_pubsub_max_in_buffer_capacity = (50 * 1024).to_string();
+    let default_rpc_pubsub_max_out_buffer_capacity = (15 * 1024 * 1024).to_string();
```

Comment:
Removed the variables, but did not the corresponding args since that can break many validator startup scripts.
We might want to print a deprecation warning/message if any of the values are set, but I did not find any elegant way of doing so via `clap` crate. Still can add the warnings if neccessary.